test/steward_benefactor_withdrawal.js

```diff
@@ -170,7 +167,7 @@ contract("WildcardSteward Benefactor collection", (accounts) => {
   // });
 
   describe("steward: benefactor withdrawal with token foreclosure", async () => {
-    it("steward: benefactor withdrawal. A token is owned for 20 minutes, but forecloses after 10 minutes. The organisation withdraws their funds after 20 minutes.", async () => {
+    it("steward: benefactor withdrawal. A token is owned for 20 minutes, but forecloses after 10 minutes. The organisation withdraws their funds after 20 minutes. [ @skip-on-coverage ]", async () => {
```

Comment:
It seems like this `@skip-on-coverage` thing isn't actually working ...

Meh, I guess we don't need it now